gtwrap/pybind_wrapper.py

```diff
@@ -193,7 +210,8 @@ def wrap_properties(self, properties, cpp_class, prefix='\n' + ' ' * 8):
             res += ('{prefix}.def_{property}("{property_name}", '
                     '&{cpp_class}::{property_name})'.format(
                         prefix=prefix,
-                        property="readonly" if prop.ctype.is_const else "readwrite",
+                        property="readonly"
+                        if prop.ctype.is_const else "readwrite",
```

Comment:
bad autoformat

Reply:
Is it? `yapf` gives me this.

Reply:
Oh sorry, I just thought this might have been an unintentional oversight, i.e. a byproduct of running autoformat on the entire document and then committing the entire file instead of (a) only autoformatting the lines you changed/added or (b) checking lines/chunks before committing, but if this was intentional and you actually prefer it this way then that's totally fine.

To elaborate on what I meant by this, is that the default autoformat is not (imo) human-friendly so we (you) might consider modifying it to be more human-friendly.  The way it is currently autoformatted is, imo, misleading since every other line starts with a kwarg, except this one with an `if` statement which is confusing both because it is on a separate line from the "then" value and because it's not indented differently.  This would be like formatting c++ code as
```c++
function(variableA,
         variableB,
         variableC
         ? variableD : variableE,
         variableF
)
```

Perhaps if you e.g. added parenthesis:
```python
                        property=("readonly"
                                  if prop.ctype.is_const else "readwrite")
```
so that the autoformatter would keep a not-misleading format?  Or change column width to 120 or something: `--style={ column_limit: 120 }`.  Totally up to you - I just thought these formatting changes were unintentional.

Reply:
Oh sorry, I just thought this might have been an unintentional oversight, i.e. a byproduct of running autoformat on the entire document and then committing the entire file instead of (a) only autoformatting the lines you changed/added or (b) checking lines/chunks before committing, but if this was intentional and you actually prefer it this way then that's totally fine.

To elaborate on what I meant by this, is that the default autoformat is not (imo) human-friendly so we (you) might consider modifying it to be more human-friendly.  The way it is currently autoformatted is, imo, misleading since every other line starts with a kwarg, except this one with an `if` statement which is confusing both because it is on a separate line from the "then" value and because it's not indented differently.  This would be like formatting c++ code as
```c++
function(variableA,
         variableB,
         variableC
         ? variableD : variableE,
         variableF
)
```

Perhaps if you e.g. added parenthesis:
```python
                        property=("readonly"
                                  if prop.ctype.is_const else "readwrite")
```
so that the autoformatter would keep a not-misleading format?  Or change column width to 120 or something: `--style={ column_limit: 120 }`.  Totally up to you - I just thought these formatting changes were unintentional.

Reply:
So that behavior is intentional by the formatter so it brings attention to the fact that the previous kwarg is conditioned on some check.

Also, your settings might explain why I sometimes get more formatting than expected. I generally like 80 width and update my code to fit that but if 120 is more convenient/pragmatic we **all** should switch and be consistent.